.github/workflows/kiali-ci.yml

```diff
@@ -27,12 +25,6 @@ jobs:
       - name: Lint Install
         run: make lint-install
 
-      - name: Gofmt
```

Comment:
Is the "gofmt" included in the "goimports" linter?
I reviewed the discussion but not sure if that was commented on before.

Reply:
> Is the "gofmt" included in the "goimports" linter?
> I reviewed the discussion but not sure if that was commented on before.

Yep! From the [goimports docs](https://pkg.go.dev/golang.org/x/tools/cmd/goimports#pkg-overview):

> In addition to fixing imports, goimports also formats your code in the same style as gofmt so it can be used as a replacement for your editor's gofmt-on-save hook.